solutions/devicesimulation-nohub/single-vm/docker-compose.yml

```diff
@@ -152,3 +152,19 @@ services:
       options:
         max-size: "10240k"
         max-file: "10"
+
+  iothubmanager-svc:
+    image: azureiotpcs/iothub-manager-dotnet:testing
```

Comment:
Since this will require merging the IoTHub manager's code and push a new docker container with 'testing' tag, I would suggest you use a different tag name here for the moment.